amnestywebsite/humanity-theme
# Adds 66/33 text with image block pattern

Ref: https://github.com/amnestywebsite/humanity-theme/issues/173

Adds 66/33 text with image block pattern

**Steps to test**:
1. Edit a post
2. Open block inserter and check new patterns is present under patterns tab
3. Insert the pattern and check the matches the example in the ticket

Example: http://bigbite.im/i/Go2keP
